src/Exception/Renderer/InlineRenderer.php

```diff
@@ -25,22 +25,22 @@
 class InlineRenderer implements RendererInterface
 {
     /**
-     * @var EngineInterface
+     * @var string
      */
-    protected $templating;
+    protected $template;
 
     /**
-     * @var string
+     * @var Environment
      */
-    protected $template;
+    private $twig;
 
     /**
-     * @param EngineInterface $templating Templating engine
-     * @param string          $template   Template to render
+     * @param Environment $twig     Twig
+     * @param string      $template Template to render
      */
-    public function __construct(EngineInterface $templating, $template)
+    public function __construct(Environment $twig, $template)
```

Comment:
Can you replace parameters with typehint and remove the comment?

Reply:
Fixed. It is `public function __construct(Environment $twig, string $template)` now without phpdoc comment